crates/symbolicator-service/tests/integration/snapshots/integration__sourcemap__sourcemap_embedded_source_expansion.snap

```diff
@@ -30,6 +31,7 @@ raw_stacktraces:
         abs_path: "http://example.com/index.html"
         lineno: 283
         colno: 17
+        in_app: false
```

Comment:
I believe this already exists like this in the test?

Reply:
Yes, the input frame in the test has this set to `false` and this is passed through to the result.